packages/core-components/src/components/Link/Link.tsx

```diff
@@ -194,9 +203,17 @@ export const Link = React.forwardRef<any, LinkProps>(
         className={classnames(classes.externalLink, props.className)}
       >
         {props.children}
-        <Typography component="span" className={classes.visuallyHidden}>
-          , Opens in a new window
-        </Typography>
+        {newWindow && (
+          <>
```

Comment:
I feel like this component is used in a lot of other places not just inline text links. I think that it could wrap other components in the future too, so maybe it's best to check that children is either text only, or that we have a seperate wrapper that's not in here for inline links like in the `EntityCard` etc.

Reply:
Do we just want to display the icon when it's a text link? Or when there's any text in the link? My thinking was that there are places that use the `Typography` components which _I think_ aren't considered just text at run time.

I like the idea of a separate `TextLink` component for just text links, that would have this additional icon and be more declarative. Is there guidance from Backstage design of whether icons should be clickable in links? ie in the MenuBar they are but in the `EntityLinksCard` they're not. I could see value in having the external icon in the menu bar if someone added a support channel icon or something.

Reply:
> My thinking was that there are places that use the Typography components which I think aren't considered just text at run time. 

No yeah, that's gonna get pretty tough I think.

> I like the idea of a separate TextLink component for just text links

Yep - me too I think.

> Is there guidance from Backstage design of whether icons should be clickable in links? 

I will try and get some more info on this!

Reply:
Any updates on this? Waiting on the design feedback before I jump into a `TextLink` component.

Reply:
Nope - not as yet, I will chase next week!